pkg/apis/eksctl.io/v1alpha5/gpu_validation_test.go

```diff
@@ -21,7 +21,7 @@ var _ = Describe("GPU instance support", func() {
 	assertValidationError := func(e gpuInstanceEntry, err error) {
 		if e.expectUnsupportedErr {
 			Expect(err).To(HaveOccurred())
-			Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("Inferentia instance types are not supported for %s", e.amiFamily))))
+			Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("%s instance types are not supported for %s", e.gpuInstanceType, e.amiFamily))))
```

Comment:
this is failing https://github.com/weaveworks/eksctl/actions/runs/3448543804/jobs/5778541223
needs to be modified to match the validation errors

Reply:
ah, yeah I meant to fix that.  I'll fix and push up another commit.